src/resources/js/validation.js

```diff
@@ -37,12 +37,250 @@ tribe.validation = {};
+			}
+
+			// Applies the type of validation
+			if ( type && _.isFunction( obj.parseType[ type ] ) ) {
+				constraint = obj.parseType[ type ]( constraint, $constraint, $field );
+				value = obj.parseType[ type ]( value, $constraint, $field );
+			}
+
+			return constraint < value;
+		},
+		isGreaterOrEqualTo: function( value, constraint, $field ) {
+			var type = $field.data( 'validationType' );
+			var $constraint = null;
+
+			// If it's not Numeric we treat it like a Selector
+			if ( ! _.isNumber( constraint ) ) {
+				$constraint = $( constraint );
+				constraint = $constraint.val();
+			}
+
+			// Applies the type of validation
+			if ( type && _.isFunction( obj.parseType[ type ] ) ) {
+				constraint = obj.parseType[ type ]( constraint, $constraint, $field );
+				value = obj.parseType[ type ]( value, $constraint, $field );
+			}
+
+			return constraint <= value;
+		},
+		isLessThan: function( value, constraint, $field ) {
+			var type = $field.data( 'validationType' );
+			var $constraint = null;
+
+			// If it's not Numeric we treat it like a Selector
+			if ( ! _.isNumber( constraint ) ) {
+				$constraint = $( constraint );
+				constraint = $constraint.val();
+			}
+
+			// Applies the type of validation
+			if ( type && _.isFunction( obj.parseType[ type ] ) ) {
+				constraint = obj.parseType[ type ]( constraint, $constraint, $field );
+				value = obj.parseType[ type ]( value, $constraint, $field );
+			}
+
+			return constraint > value;
+		},
+		isLessOrEqualTo: function( value, constraint, $field ) {
+			var type = $field.data( 'validationType' );
+			var $constraint = null;
+
+			// If it's not Numeric we treat it like a Selector
+			if ( ! _.isNumber( constraint ) ) {
+				$constraint = $( constraint );
+				constraint = $constraint.val();
+			}
+
+			// Applies the type of validation
+			if ( type && _.isFunction( obj.parseType[ type ] ) ) {
+				constraint = obj.parseType[ type ]( constraint, $constraint, $field );
+				value = obj.parseType[ type ]( value, $constraint, $field );
+			}
+
+			return constraint >= value;
+		},
+		isEqualTo: function( value, constraint, $field ) {
+			var type = $field.data( 'validationType' );
+			var $constraint = null;
+
+			// If it's not Numeric we treat it like a Selector
+			if ( ! _.isNumber( constraint ) ) {
+				$constraint = $( constraint );
+				constraint = $constraint.val();
+			}
+
+			// Applies the type of validation
+			if ( type && _.isFunction( obj.parseType[ type ] ) ) {
+				constraint = obj.parseType[ type ]( constraint, $constraint, $field );
+				value = obj.parseType[ type ]( value, $constraint, $field );
+			}
+
+			return constraint == value;
+		},
+		isNotEqualTo: function( value, constraint, $field ) {
+			var type = $field.data( 'validationType' );
+			var $constraint = null;
+
+			// If it's not Numeric we treat it like a Selector
+			if ( ! _.isNumber( constraint ) ) {
+				$constraint = $( constraint );
+				constraint = $constraint.val();
```

Comment:
I'd stick a check in here just in case we get a garbage selector - either bail or treat as 0? 
Probably before the `val()`
Same for the previous methods.